src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -106,6 +102,9 @@ protected override void RegisterOnAfterPackageLoadedAsyncWork(PackageLoadTasks a
 
         Task OnAfterPackageLoadedBackgroundThreadAsync(PackageLoadTasks afterPackageLoadedTasks, CancellationToken cancellationToken)
         {
+            // Ensure the options persisters are loaded since we have to fetch options from the shell
+            _ = ComponentModel.GetService<IGlobalOptionService>();
```

Comment:
>_ = ComponentModel.GetService<IGlobalOptionService>(); [](http://example.com/codeflow?start=12&length=54)

I debated just not having this, but I wanted to keep it similar  to before in that it explicitly ensures the service is at least created by this point.

Reply:
At this point we're not eagerly creating any persisters, right? It's just creating the service, but the persisters themselves are still lazy? In that case, I'd toss this. It's not doing anything (but looks like it might be.)